common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -3852,12 +3858,24 @@
     ],
     "sqlState" : "42802"
   },
+  "STATE_STORE_INVALID_CONFIG_AFTER_RESTART" : {
+    "message" : [
+      "<configName> <oldConfig> is not equal to <newConfig>. Please set <configName> to <oldConfig>, or restart with a new checkpoint directory."
```

Comment:
Could you paste an example of what this message looks like ?

Reply:
```
org.apache.spark.sql.execution.streaming.state.StateStoreInvalidConfigAfterRestart: [STATE_STORE_INVALID_CONFIG_AFTER_RESTART] timeMode NoTime is not equal to ProcessingTime. Please set timeMode to NoTime, or restart with a new checkpoint directory. SQLSTATE: 42K06
	at org.apache.spark.sql.execution.streaming.state.StateStoreErrors$.invalidConfigChangedAfterRestart(StateStoreErrors.scala:180)
	at org.apache.spark.sql.execution.streaming.TransformWithStateOperatorProperties$.validateOperatorProperties(TransformWithStateVariableUtils.scala:113)
	at org.apache.spark.sql.execution.streaming.TransformWithStateExec.validateNewMetadata(TransformWithStateExec.scala:477)
	at org.apache.spark.sql.execution.streaming.IncrementalExecution$StateSchemaAndOperatorMetadataRule$$anonfun$5.applyOrElse(IncrementalExecution.scala:222)
	at org.apache.spark.sql.execution.streaming.IncrementalExecution$StateSchemaAndOperatorMetadataRule$$anonfun$5.applyOrElse(IncrementalExecution.scala:199)
	at org.apache.spark.sql.catalyst.trees.TreeNode.$anonfun$transformDownWithPruning$1(TreeNode.scala:470)
	at org.apache.spark.sql.catalyst.trees.CurrentOrigin$.withOrigin(origin.scala:84)
```

Reply:
Could we update as `Config <configName>=<oldConfig> is not equal to` ?